aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -308,6 +305,10 @@ static Tensor _mkldnn_convolution(
     output.resize_(output_sizes, memory_format);
     y = itensor_from_tensor(output);
   }
+  if (mkldnn_conv_enabled_fpmath_mode_bf16() &&
+      input_t.scalar_type() == at::kFloat) {
+    op_attr.set_fpmath_mode(dnnl_fpmath_mode_bf16);
```

Comment:
This changes the `op_attr` state? How can it still be a const ref?

Reply:
It's not a const ref here. `_mkldnn_convolution` create `op_attr` and also set the post op here:  https://github.com/pytorch/pytorch/pull/126050/files#diff-f77e0c21c1a0a2ae4a2cc5e9bf3c7747241e294eba13402b96818a668cfb3678R233-R238. I also set the `fp_math_mode`.

In test next level function `_mkldnn_convolution_out` it's a const ref. https://github.com/pytorch/pytorch/pull/126050/files#diff-f77e0c21c1a0a2ae4a2cc5e9bf3c7747241e294eba13402b96818a668cfb3678L176